src/software/geom/polynomial.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include <cmath>
+#include <vector>
+
+#include "software/geom/point.h"
+
+/**
+ * Polynomial is a representation of a polynomial
+ * of the form coeffs[0] + coeffs[1] * + coeff[2] * x^2...
```

Comment:
I think you're missing an X^1 here

Reply:
done

Reply:
I don't see any change, did you push? @jonathanlew